sunspot/lib/sunspot/session.rb

```diff
@@ -100,6 +100,22 @@ def index!(*objects)
     end
 
     #
+    # See Sunspot.atomic_update
+    #
+    def atomic_update(clazz, updates = {})
```

Comment:
I've added `integration\atomic_updates_spec.rb` is that what you were asking for?